adding keypoints to 02 (#400)
datacarpentry/python-ecology-lesson: _episodes/02-starting-with-data.md

    - "Perform basic mathematical operations and summary statistics on data in a Pandas DataFrame."
    - "Create simple plots."
keypoints:

>     - "Libraries enable you to add functions to Python. Pandas is a popular library for working with data."

Reviewer: I suggest splitting this into two:
1. Python libraries extend the functionality of Python.
2.

Reviewer: ...Classic mistake (hitting Return before completing the message)...
2. Pandas is a popular Python library for ....

Author: ```suggestion
    - "Libraries enable you to add functions to Python." 
    - "Pandas is a popular library for working with data."
```

    - "Perform basic mathematical operations and summary statistics on data in a Pandas DataFrame."
    - "Create simple plots."
keypoints:

>     - "Libraries enable you to add functions to Python." 

Reviewer: I'm not sure we should say that libraries enable us to add functions... They allow us extend the limits of what Python can do...

Author: Oh, that's a good point - you could add your own functions without installing any libraries.

Author: ```suggestion
    - "Libraries enable us to extend the functionality of Python." 
```

Author: Maybe that?

Reviewer: 👍

    - "Pandas is a popular library for working with data."

>     - "A DataFrame can be used to access data either by column name, column integer, or row."

Reviewer: Dataframe is a Pandas data structure that allows one to access data by column (name or index) or row.

Author: ```suggestion
    - "A Dataframe is a Pandas data structure that allows one to access data by column (name or index) or row."
```

    - "Aggregating data using the `groupby()` function enables you to generate useful summaries of data quickly."

>     - "Plots can be created from DataFrames or from subsets of data that have been generated with `groupby()`."

Reviewer: `from DataFrames or from subsets of data` -> `from DataFrames or subsets of data`

Author: ```suggestion
    - "Plots can be created from DataFrames or subsets of data that have been generated with `groupby()`."
```
